backend/src/main/java/com/benine/backend/database/MySQLDatabase.java

```diff
@@ -245,6 +246,71 @@ public void useDatabase() throws SQLException {
     }
   }
 
+  @Override
+  public void addTag(String name) throws SQLException {
+    Statement statement = null;
+    try {
+      statement = connection.createStatement();
+      final String sql = String.format("INSERT INTO presetsdatabase.tag VALUES(%s)",
+          name);
+      statement.executeUpdate(sql);
+      statement.close();
+    } catch (SQLException e) {
+      e.printStackTrace();
+      getLogger().log("Tag couldn't be added", LogEvent.Type.CRITICAL);
+    } finally {
+      if (statement != null) {
+        try {
+          statement.close();
+        } catch (SQLException e) {
+          e.printStackTrace();
+        }
+      }
+    }
+  }
+
+  @Override
+  public void deleteTag(String name) throws SQLException {
+    Statement statement = null;
+    try {
+      statement = connection.createStatement();
+      String sql = "DELETE FROM tag WHERE name = " + name;
+      statement.executeUpdate(sql);
+      statement.close();
+    } catch (SQLException e) {
+      e.printStackTrace();
+      getLogger().log("Tag couldn't be deleted.", LogEvent.Type.CRITICAL);
+    } finally {
+      if (statement != null) {
```

Comment:
This is duplicate code which could be in a different method. Also, please don't print the stack trace but use the logger instead.